### DISCUSSION TITLE
Removing the minimum limit in the camera distance value in the camera-orbit attribute

### DISCUSSION THREAD

**Discussion by antekj** [google/model-viewer#1938] [Q&A]
Hi,
I would like to mimic a VR experience by being able to enter the mesh of a placed 3d model.
In my understanding, this could be easily achievable by removing the bottom limit of the camera distance value within the camera-orbit attribute that currently prevents from entering the mesh. 
Without such limitation, I imagine it would be possible to place the camera at the camera-target position by specifying e.g. camera-orbit="45deg 55deg 0m", right?
I would be very thankful for a little hint re where to tweak the .js code to remove the limit, it seems like a simple operation.
Thanks!

**Comment by antekj** [comment]
As I think more about it, even if it's possible to remove that limitation in the model-viewer.js, the AR Quick Look will still prevent from placing a large-scale object around the user, right? What I need to make is a 3d tent which you can enter.

**Comment by elalish** [comment]
If you're talking about AR, there is no camera limit and the mesh can be entered. If you mean the plain 3D mode, you can simply set https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-minCameraOrbit to `auto auto 0m`.

**Comment by antekj** [comment]
Thank you @elalish ! That indeed solves the problem in the plain 3D mode, but in WebXR and Scene Viewer, no matter how big your model is and how close to yourself you spawn it in AR, it always gets spawned in front of the user, with the origin even tens of meters away from the user. iOS's Quick Look doesn't do that though, it spawns the model with its origin right where you're pointing, so with a large model you end up inside the mesh right away.
Do you think it would be possible to follow Quick Look's behaviour in that regard for WebXR and Scene Viewer?

Also, on a side note, is it possible to tell the Scene Viewer to disable occlusion?

Thank you again @elalish

**Comment by elalish** [comment]
Ah, so your question is actually how to control where a model is initially spawned in AR. I don't believe that's controllable in SceneViewer, but we could consider it in webXR mode. Could you share some examples so I can get a better idea of the user experience you're trying to create? As for occlusion, see #1733.

**Comment by antekj** [comment]
Thank you for your quick reply @elalish . Sure, here is a little test: https://unicef-ar.org/test/ 
It is a room around 9m-long and -wide. In Quick Look you end up inside right away upon spawning it. In WebXR, after spawning,  you have to drag the model toward you or walk inside it, which is actually cool but becomes a problem for users with limited space.

Re the occlusion, thanks for pointing me in the right direction.
